eventuals/control-loop.h

```diff
@@ -94,7 +86,7 @@ class ControlLoop final
     task_.Start(
         std::string(name_), // Copy.
         [borrow = std::move(borrow)]() mutable { borrow.relinquish(); },
-        [](std::exception_ptr) { LOG(FATAL) << "Unreachable"; },
+        []() { LOG(FATAL) << "Unreachable"; },
```

Comment:
HOW COOL IS THAT! IF NO ERRORS ARE PROPAGATED THAN WE HAVE AN EMPTY ARGUMENT LIST! SOOOOOO COOOOOL! 💪